### ISSUE TITLE
Define custom loss function

### ISSUE THREAD

**Issue by zouqian2468** [Azure/mmlspark#871]
How can I define custom loss function on the scala version of lightGBM?

**Comment by imatiach-msft** [comment]
@zouqian2468 this isn't available yet, will need to look into this more

**Comment by robert-howley-zocdoc** [comment]
+1 on this idea. would be a very helpful feature

**Comment by imatiach-msft** [comment]
closing issue as this has been implemented in PR:
https://github.com/Azure/mmlspark/pull/1054
relevant issues:
https://github.com/Azure/mmlspark/issues/873, https://github.com/Azure/mmlspark/issues/871, https://github.com/Azure/mmlspark/issues/433
